src/index.js

```diff
@@ -6,46 +6,52 @@ const tags = tagsW3.map(tag => tag.replace(/[<|>]/g, ""));
 
 // Wait for the DOM to be fully loaded
 document.addEventListener("DOMContentLoaded", function () {
-    uploadTagsFromLocalStorage(); // list all tags in localStorage
-    updateProgressBar();
-    // Find the form element and add a submit event listener
-    const form = document.querySelector("form");
-    form.addEventListener("submit", function (event) {
-        // on submit
-        event.preventDefault(); // Prevent the default form submission
-        submitVal(); // call our submitVal function
-    });
 
-    function submitVal() {
-        let val = document.getElementById("input1").value;
-        // remove all white spaces and non-alphabetical characters except . ! - 
-        let regex = /[^a-zA-Z!-.]/g;
-        // change input to lower case and apply regex
-        let sanitizedVal = val.toLowerCase().replace(regex, "");
+  uploadTagsFromLocalStorage(); // list all tags in localStorage
+  updateProgressBar();
+  // Find the form element and add a submit event listener
+  const form = document.querySelector("form");
+  form.addEventListener("submit", function (event) {
+    // on submit
+    event.preventDefault(); // Prevent the default form submission
+    submitVal(); // call our submitVal function
+  });
 
-        let li = "<li>" + sanitizedVal + "</li>";
-        // check if the input value is in the tags array
-        let found = tags.some(tag => tag.toLowerCase() === sanitizedVal);
-        if (found) {
-            document.getElementById("list").insertAdjacentHTML("beforeend", li);
-            saveTagToLocalStorage(sanitizedVal); // Save tag to localStorage
-            updateProgressBar();
-        }
-        document.getElementById("input1").value = "";
+  function submitVal() {
+    let val = document.getElementById("input1").value;
+    // remove all white spaces and non-alphabetical characters except . ! - 
+    let regex = /[^a-zA-Z!-.]/g;
+    // change input to lower case and apply regex
+    let sanitizedVal = val.toLowerCase().replace(regex, "");
+    let li = "<li>" + sanitizedVal + "</li>";
+    // check if the input value is in the tags array
+    let found = tags.some(tag => tag.toLowerCase() === sanitizedVal);
+    // get the list of inputs already made
+    const getList = localStorage.getItem("myTags");
```

Comment:
I guess it makes sense for the refactor issue to also check if usage of `const` or `let` is appropriate 🤔 
I understand that `const` holds a value which never changes, so I guess I should use `let` here since the list is updating, even though it updates within its declaration?!

What do you think about writing const values in uppercase? I often read that this is how you do it (maybe only in other languages?!)

Reply:
It certainly is a good idea to use `const` wherever possible. (I struggle with this, my instinct is always to pick `let`)

Regarding the uppercase:
In JavaScript, the `const` keyword means that the variable cannot be reassigned after its initial assignment. 

So, if `const name = “John”`, we can’t later have `name =”Jane”` (but we can do this with let!)

The convention of using uppercase for const variables applies to values that are meant to be constants in a more "global" or widely-used sense. It could be an API key, or the number of days in a year, that you need to use throughout your program, and so create a variable that by its name is self explanatory and using `const` you make sure it cannot be changed by mistake. They are often referred to as "magic numbers" or "magic strings". 

These are values that won't change throughout the program's execution and are used in multiple places, not in just one block or module, but across the application.

Examples:
const API_KEY = ….
const PI = 3.14159;